packages/web/src/components/artist/ArtistSupporting.tsx

```diff
@@ -55,8 +55,19 @@ export const ArtistSupporting = (props: ArtistSupportingProps) => {
     return rankedSupportingList
       .sort((s1, s2) => s1.rank - s2.rank)
       .map(s => usersMap[s.receiver_id])
+      .filter(Boolean)
   })
 
+  /**
+   * It's possible that we don't have the data for which artists
+   * this artist is supporting. Thus, we fetch in this case.
+   */
+  useEffect(() => {
+    if (supportingForArtistIds.length === 0) {
+      dispatch(fetchSupportingForUser({ userId: user_id }))
+    }
```

Comment:
Is it possible this artist simply has 0 supporting? Do we want to refetch every single time we hover over the popover?

Reply:
good call. fixed thx